week2/homework/src/index.js

```diff
@@ -1,3 +1,70 @@
 'use strict';
 
-// TODO: Write the homework code in this file
+'use strict';
+
+const { appendFile, readFile, writeFile } = require('fs');
+const { promisify } = require('util');
+const appendFileWithPromise = promisify(appendFile);
+const readFileWithPromise = promisify(readFile);
+const writeFileWithPromise = promisify(writeFile);
+const TODO_FILE = 'todo.json';
+
+async function main() {
+  const [, , cmd, ...args] = process.argv;
+
+  switch (cmd) {
+    case 'help':
+    default:
+      console.log(`
+      options : 
+      1) help : show help section.
+      2) add : add a todo item. (node . add "")
+      3) list: show current todo's list.(node . list)
+      4) remove: remove a todo item (node . remove "the index number")
+      5) reset: remove all items from the list. (node . reset)
+      6) update: update any item from todo list with a new item (node . update "new item")
+`);
+      break;
+    case 'add': {
+      const data = await readFileWithPromise(TODO_FILE, 'utf8').catch(() => '[]');
+      const todos = JSON.parse(data);
```

Comment:
You could put those 2 lines in a function that you can reuse in your other cases.